src/constants/date.js

```diff
@@ -1,4 +1,10 @@
 export const TYPE_DATE = "date";
-export const TYPE_WEEKDAY = "wday";
+export const TYPE_WEEKDAY = "day";
```

Comment:
```suggestion
export const TYPE_WEEKDAY = "wDay";
```
day로 혹시 변경하신 이유가 있을까요?